clusterloader2/docs/test_suite_design.md

```diff
@@ -0,0 +1,43 @@
+# Cluster loader - TestSuite
+
+## Background
+
+Since 08/2018 cluster loader become main tool for kubernetes e2e performance tests.
+It is a very convenient tool for running single test case, but not for running
+test bundles. Currently it is impossible to run the same test
+with two different parameter sets in a single job run.
+This forces developers to create separate job configs for
+almost each test.
+
+## TestSuite
+
+TestSuite is a set of test scenarios running in a single test job.
+
+```
+type TestSuite []TestScenario
+```
+
+## Test
+
+TestScenario is a representation of single test scenario.
```

Comment:
This sentence also doesn't bring any value :)

Reply:
I changed structure of the document a little bit. I moved parts of the description to the corresponding fields.